src/character.h

```diff
@@ -597,6 +598,8 @@ class Character : public Creature, public visitable<Character>
         // turn the character expired, if -1 it has not been set yet.
         int turn_died = -1;
 
+        mutable pathfinding_settings path_settings;
```

Comment:
I intended it to be a sort of a cache, to avoid rebuilding the same data every time.
`mutable` because some minor elements of it do change - bash strength of an actor in case of NPCs.